skymp5-functions-lib/src/mpApiInteractor.ts

```diff
@@ -263,9 +276,10 @@ export class MpApiInteractor {
         CounterProperty.set(actorId, counter, current + (by ?? 0));
         return current;
       },
-      getSetting(name: string): any {
-        return mp.getServerSettings()[name]
-      }
-    }
+      getServerSetting(name: string): any {
+        // Add typings
```

Comment:
Maybe TODO then? Ideally `TODO(#xyzw): ...` where #xyzw is an issue id, but might be just TODO

Reply:
It's properly written in merge commit